src/libc.cr

```diff
@@ -1,29 +1,32 @@
 lib LibC
+  ifdef x86_64
+    alias SizeT = UInt64
+    alias SSizeT = Int64
+  else
+    alias SizeT = UInt32
+    alias SSizeT = Int32
+  end
+
+  #ifdef windows
```

Comment:
Crystal currently doesn't support Windows. This is to ease work for someone who would be implementing Windows support in the future.